### DISCUSSION TITLE
I want to cancel the agreement and get my money back

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37911] [New to GitHub] [answered]
Hi 

You provide me with the payment automatically without my knowledge and I would like to cancel the agreement and get my money back

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mohtrefiptv! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) The Community is unable to assist in matters of payment and account-specifics so I'll be locking the thread.
